src/utils/revisionDraftUtils.ts

```diff
@@ -26,3 +34,43 @@ export function getParentResourceFromItem(item: ContainerAppItem | RevisionsItem
         return item.revision;
     }
 }
+
+/**
+ * An informational deploy pop-up to show after executing a revision draft command
+ */
+export async function showRevisionDraftDeployPopup(context: IActionContext, containerApp: ContainerAppModel): Promise<void> {
+    if (!await settingUtils.getGlobalSetting('showDraftCommandDeployPopup')) {
+        return;
+    }
+
+    const yes: string = localize('yes', 'Yes');
+    const no: string = localize('no', 'No');
+    const dontShowAgain: string = localize('dontShowAgain', 'Don\'t show again');
+
+    const message: string = localize('message', 'Would you like to deploy these changes? Click "Yes" to proceed, or "No" to continue making changes.');
```

Comment:
I think we should definitely run through this with Anthony/Misty to finalize the wording.